profiling-replayer/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "datadog-profiling-replayer"
+authors = ["Levi Morrison <levi.morrison@datadoghq.com>"]
+description = "Takes a pprof file and 'replays' it using libdatadog commands."
+version = "3.0.0"
```

Comment:
why 3.0?

Reply:
It matches the version field of `profiling/Cargo.toml`.